bids/variables/io.py

```diff
@@ -171,7 +173,11 @@ def _load_time_variables(layout, dataset=None, columns=None, scan_length=None,
 
         # Process event files
         if events:
-            dfs = listify(layout.get_events(img_f, derivatives=derivatives))
+            domains = {'ignore': 'bids', 'only': 'derivatives',
+                      'both': ['bids', 'derivatives']}[derivatives]
+            dfs = layout._get_nearest_helper(img_f, '.tsv', type='events',
```

Comment:
Still not working for me, but I think the issue is not with grabbit, but with this line. This only returns the file in the main project no matter what I do.

This line works though: `layout.get_nearest(img_f, type='events', ignore_strict_entities=['type'], full_search=True, all_=True)`

What's the rationale for using the helper, vs get_nearest?